device/arista/x86_64-arista_7800r3_48cqm2_lc/platform.json

```diff
@@ -1,28 +1,44 @@
 {
     "chassis": {
         "name": "7800R3-48CQM2-LC",
-        "components": [],
+        "components": [
+            {
+                "name": "Aboot()"
+            },
+            {
+                "name": "Scd(addr=0000:00:18.7)"
+            },
+            {
+                "name": "Scd(addr=0000:08:00.0)"
```

Comment:
Not all linecards are the same. Here the value for this line needs to be `Scd(addr=0000:07:00.0)`